### PR TITLE
Wire Sidebar for showing extra info panels

### PR DESCRIPTION
**Pull request by danieledamiani**

### CODE REVIEW

File: packages/tempus-client_v2/src/components/main/Main.tsx [hunk 1]
```diff
@@ -1,28 +1,40 @@
...
+      <Sidebar initialView="Deposit" onSelectedView={setSelectedView} />
       <div className="tc__dashboard">
         <div className="tc__dashboard__row">
           <Pool language={language} />
           <Term language={language} />
-          <CurrentPosition language={language} />
-        </div>
-        <div className="tc__dashboard__row">
-          <ProfitLoss language={language} />
+          {showExtraInfo && <CurrentPosition language={language} />}
```
Comment (lazarmitic) [inline packages/tempus-client_v2/src/components/main/Main.tsx:35]: I think all views should show current position as long as user has some balance in the pool.

File: packages/tempus-client_v2/src/components/main/Main.tsx [hunk 2]
```diff
@@ -1,28 +1,40 @@
...
         </div>
+        <div className="tc__dashboard__row">{showExtraInfo && <ProfitLoss language={language} />}</div>
```
Comment (lazarmitic) [inline packages/tempus-client_v2/src/components/main/Main.tsx:37]: Same for ProfitLoss.

File: packages/tempus-client_v2/src/components/sidebar/Sidebar.tsx [hunk 3]
```diff
@@ -1,19 +1,46 @@
...
+  initialView: string;
+};
+
+type SidebarProps = SidebarInProps & SidebarOutProps;
+
+const Sidebar: FC<SidebarProps> = ({ initialView, onSelectedView }) => {
+  const [selectedView, setSelectedView] = useState<string>('');
+
+  const onItemClick = useCallback(
+    (itemName: string) => {
```
Comment (lazarmitic) [inline packages/tempus-client_v2/src/components/sidebar/Sidebar.tsx:32]: Change type here to transaction view

File: packages/tempus-client_v2/src/components/sidebar/Sidebar.tsx [hunk 4]
```diff
@@ -1,19 +1,46 @@
...
+};
+
+type SidebarInProps = {
+  initialView: string;
+};
+
+type SidebarProps = SidebarInProps & SidebarOutProps;
+
+const Sidebar: FC<SidebarProps> = ({ initialView, onSelectedView }) => {
+  const [selectedView, setSelectedView] = useState<string>('');
```
Comment (lazarmitic) [inline packages/tempus-client_v2/src/components/sidebar/Sidebar.tsx:29]: This should be initialized with inital iew prop

File: packages/tempus-client_v2/src/components/main/Main.tsx (hunk 1 above)
Comment (danieledamiani) [reply packages/tempus-client_v2/src/components/main/Main.tsx:35]: In the design `Deposit` and `Mint` don't show it, I'll double check it

File: packages/tempus-client_v2/src/components/main/Main.tsx (hunk 2 above)
Comment (danieledamiani) [reply packages/tempus-client_v2/src/components/main/Main.tsx:37]: same as above

File: packages/tempus-client_v2/src/components/sidebar/Sidebar.tsx (hunk 3 above)
Comment (danieledamiani) [reply packages/tempus-client_v2/src/components/sidebar/Sidebar.tsx:32]: fixed

File: packages/tempus-client_v2/src/components/sidebar/Sidebar.tsx (hunk 4 above)
Comment (danieledamiani) [reply packages/tempus-client_v2/src/components/sidebar/Sidebar.tsx:29]: when `useState` fires the props may be not available

File: packages/tempus-client_v2/src/components/main/Main.tsx (hunk 1 above)
Comment (danieledamiani) [reply packages/tempus-client_v2/src/components/main/Main.tsx]: checked, they're shown if the user has assets. Added a TODO

**Comment by danieledamiani** [comment]
build is failing but it's expected

**Pull request merged**